### PR TITLE
Admin UI: Adding h1 page headings for screen reader navigation

### PR DESCRIPTION
**Pull request by coder-karen**
This partially addresses issues mentioned in #6533

#### Changes proposed in this Pull Request:

* In this PR, h1 headings have been added to the following admin pages:
  * `wp-admin/admin.php?page=jetpack#/dashboard`
  * `wp-admin/admin.php?page=jetpack#/my-plan`
  * `wp-admin/admin.php?page=jetpack#/recommendations/summary`
  * `wp-admin/admin.php?page=jetpack#/security`
  * `wp-admin/admin.php?page=jetpack#/performance`
  * `wp-admin/admin.php?page=jetpack#/writing`
  * `wp-admin/admin.php?page=jetpack#/sharing`
  * `wp-admin/admin.php?page=jetpack#/discussion`
  * `wp-admin/admin.php?page=jetpack#/traffic`
  * `wp-admin/admin.php?page=stats`
  * `wp-admin/admin.php?page=jetpack#/privacy`
  * `wp-admin/admin.php?page=jetpack_about`
  * `wp-admin/admin.php?page=jetpack_modules`

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack Product Discussion

n/a.

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Visit the listed pages with this PR checked out. You shouldn't notice any difference visually.
* Check different screen-sizes - there should still be no difference.
* Inspecting the source code, you should see an h1 tag added with an appropriate heading:

Screenshot 2022-07-04 at 09 00 44

* Design review shouldn't be needed as these elements are being hidden (except on the Site Stats page, where the h2 has been changed to a h1) - using the WordPress [screen-reader-text class ](https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/)(thank you @bluefuton for mentioning this)

### CODE REVIEW

File: projects/plugins/jetpack/_inc/client/components/admin-notices/style.scss [hunk 1]
```diff
@@ -42,4 +42,13 @@
 	.dops-notice__action.notice-dismiss {
 		height: 100%;
 	}
+
+	.jp-heading-sr-only {
```
Comment (bluefuton) [inline projects/plugins/jetpack/_inc/client/components/admin-notices/style.scss]: Do we need to add a new class for this or can we use `.screen-reader-text`?

https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/

Comment (coder-karen) [reply projects/plugins/jetpack/_inc/client/components/admin-notices/style.scss]: That works perfectly! Thank you for mentioning this, I've changed all the class-names to that now.

**Comment by coder-karen** [comment]
(Ignore this comment - just testing comments on PRs over at this PR - `https://github.com/Automattic/jetpack/pull/24977`)

**Comment by coder-karen** [comment]
(Ignore this comment - testing comments as part of testing this PR: `https://github.com/Automattic/jetpack/pull/24977`. And testing labels - sorry about the label mess above)

**Review by samiff — APPROVED** [review]
Looks good 👍

**Pull request merged**